docs/Other SDL Features/Batch Sending RPCs/index.md

```diff
@@ -1,11 +1,17 @@
 # Batch Sending RPCs
 There are two ways to send multiple requests to the head unit: concurrently and sequentially. Which method you should use depends on the type of RPCs being sent. Concurrently sent requests might finish in a random order and should only be used when none of the requests in the group depend on the response of another, such as when uploading a group of artworks. Sequentially sent requests only send the next request in the group when a response has been received for the previously sent RPC. Requests should be sent sequentially when you need to know the result of a previous request before sending the next, like when sending the several different requests needed to create a menu.
```

Comment:
"Uploading a group of artworks" is no longer a valid use case for this. `Subscribing to several hard buttons` is a better use case. Our iOS docs should be updated to match the Android ones.

Reply:
changed to match